.circleci/config.yml

```diff
@@ -135,24 +138,33 @@ commands:
           command: |
             aws configure --profile sdk_s3_release set region us-east-1
             echo -e "[sdk_s3_release]\naws_access_key_id=${S3_RELEASE_AWS_ACCESS_KEY_ID}\naws_secret_access_key=${S3_RELEASE_AWS_SECRET_ACCESS_KEY}\n" >> ~/.aws/credentials
+  pre_start_simulator:
+    description: >-
+      pre start simulator, build may fail is simulator is not started
+    steps:
+      - run:   
+          name: pre-start simulator
+          command: xcrun instruments -w "iPhone 8 (10.1) [" || true    
```

Comment:
is it possible to run on the latest simulator?

Reply:
here only 10.1 and 10.2 work.

Reply:
done